.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -1,6 +1,7 @@
 name: Bug Report
 description: Let us know about an issue you experienced with this software
-# labels: ["some existing label","another one"]
+labels: [ I2-bug ]
```

Comment:
Also the `unconfirmed` maybe?

Reply:
https://github.com/paritytech/labels/blob/main/ruled_labels/specs_polkadot-sdk.yaml#L83-L85
That sounds reasonable yes.